extensions/amp-story/amp-story-animation.md

```diff
@@ -0,0 +1,648 @@
+---
+$category@: presentation
+formats:
+  - stories
+teaser:
+  text: An animation effect definition for an AMP story.
+---
+
+<!--
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-story-animation
+
+The `<amp-story-animation>` element defines a custom animation effect in an
+[`amp-story`](./amp-story.md). These effects rely on the [Web Animations API](https://www.w3.org/TR/web-animations/) to define and run animations in AMP Stories.
+
+## Example
+
+```html
```

Comment:
Could we please make this an inline executable code sample?
https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#inline-sample

Adding `orientation="portrait"` will set the preview in for portrait

Reply:
Good idea. Can I test the result locally somehow?

Reply:
On the amphtml repo, unfortunately no - unless @sebastianbenz knows something I don't.

I'm happy to test for you if needed tho!

Reply:
Unfortunately, there's no simple way. If you have a local copy of amp.dev you can temporarily add your markdown file there (somewhere under `pages/content/amp.dev/`) and run `gulp develop` to test it locally.